### PR TITLE
clarify participant-specificity

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

File: docs/schedule.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
 # OSGVS21 Schedule
 
-This page contains schedule details for **full, invited participants** of the OSG Virtual School 2021.
+This page contains schedule details for **full, invited participants** of the OSG Virtual School 2021,
+who were selected following a Spring 2021 application process.
```
Comment (osg-cat) [inline docs/schedule.md]: Except that this is not accurate for the 2020 applicants.

File: docs/schedule.md [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
-The first instance is for School participants only,
-and the second instance is for School participants and the general public who have registered in advance.
-Participants may attend one or both times.
+The first instance is for invited School participants only,
+and the second instance will also be open to the general public who have registered in advance.
+Invited participants may attend one or both times.
 
 During the times marked as &ldquo;Staff available&rdquo;,
-School staff will be monitoring email and Slack for questions.
+School staff will be monitoring email and the OSG School Slack for questions.
```
Comment (osg-cat) [inline docs/schedule.md:18]: I’m not sure why this addition is important. If I’m on Slack, I’m watching the whole thing. Also, most people reading this page will have access only to one workspace. I would omit this one.

**Review by osg-cat — CHANGES_REQUESTED** [review]
A couple places to think about some more, but the rest is all good.

File: docs/schedule.md (hunk 2 above)
Comment (lmichael107) [reply docs/schedule.md:18]: I'd personally prefer to only expect this stuff in the OSG School Slack. My addition makes things more specific/accurate, not less, and I still think it's an improvement.

File: docs/schedule.md (hunk 1 above)
Comment (lmichael107) [reply docs/schedule.md]: fixed

**Comment by lmichael107** [comment]
ping @osg-cat

**Review by osg-cat — APPROVED** [review]

**Comment by lmichael107** [comment]
@osg-cat you want to merge, now that my slides are included?

**Pull request merged**